aiohttp_apischema/generator.py

```diff
@@ -154,6 +155,8 @@ def _save_handler(self, handler: APIHandler[APIResponse[object, int]]) -> _Endpo
                 ep_data["summary"] = summary
             if desc:
                 ep_data["desc"] = desc
+            if tags:
+                ep_data["tags"] = list(tags)
```

Comment:
Ah, this won't work correctly with Iterable (as highlighted by mypy), so let's convert to list before this function call.
```suggestion
            if tags:
                ep_data["tags"] = tags
```

Reply:
Is using Collection instead ok?

Reply:
Iterable is more flexible, but doesn't really make much difference.